# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1326400688 18000
#      Thu Jan 12 15:38:08 2012 -0500
# Branch stable
# Node ID 606e634fe9511b47f0b28f4c777f75191c618f58
# Parent  64d9f33313cc8c691974bcd123357e24bccbabdc
Added tag rc-3-6-0-1 for changeset 64d9f33313cc

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -52,8 +52,9 @@ bd2643f0ce57d070963bedd48857405f6924aa85
 695141f1c05cf1b240592bdd18e7a1503bb2a539 ss-3-3-55
 901d466ee55ac902a875ec0ade6f1eccef0841dc release-3-4-1
 3666e8e6f96e6899b8306d6ea9614aadf0500d67 release-3-4-2
 b0e70a71647b671ebcfa7a79af1ae6d3c0f52065 release-3-4-3
 3781981be535e80d44c85373b8fdaa60ca5cd097 ss-3-5-90
 ff5588774680d4f54567311fc109c8e351950f1c ss-3-5-91
 a737b3fb9c4d89d3694da6b4e623aeee64b212e1 ss-3-5-92
 72aebe6196414e38ef802469ff6e238f914f04c9 rc-3-6-0-0
+64d9f33313cc8c691974bcd123357e24bccbabdc rc-3-6-0-1
